app/scripts/modules/netflix/canary/canaryConfigSelector.directive.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+let angular = require('angular');
+
+module.exports = angular
+  .module('spinnaker.netfilx.widgits.canaryConfigSelector.directive', [
+    require('./canary.read.service'),
+    require('../../core/utils/lodash')
+  ])
+  .directive('canaryConfigSelector', () => {
+    return {
+      scope: {},
+      bindToController: {
+        model: '=' ,
+        applicationName: '='
+      },
+      templateUrl: require('./canaryConfigSelector.directive.html'),
+      controllerAs: 'vm',
+      controller: function($scope, canaryReadService, _) {
+        let vm = this;
+
+        vm.selectorChanged = (model) => {
+          if (model) {
+            vm.model = model;
+            vm.model.successfulScore = model.canarySuccessCriteria.canaryResultScore;
+            vm.model.unhealthyScore = model.canaryHealthCheckHandler.minimumCanaryResultScore;
+            vm.model.resultStrategy = model.combinedCanaryResultStrategy;
+          }
+        };
+
+        vm.isNameInConfigList = (selectedName) => {
+          return getSelected(vm.canaryConfigs, selectedName).length ? true : false;
+        };
+
+        let setOptions = (results) => {
+          vm.canaryConfigs = results;
+          return vm.canaryConfigs;
+        };
+
+        let getSelected = _.curry((canaryConfigs = [], canaryName) => {
+          return canaryConfigs.filter((config) => config.name === canaryName);
+        });
```

Comment:
Does `curry` do anything that arrow functions won't? e.g.
```javascript
let getSelected = (canaryConfigs = [], canaryName) => {
  return () => canaryConfigs.filter((config) => config.name === canaryName);
});
```
Not against it, just wondering, as I haven't used lodash's `curry` before. I kinda like that it makes it pretty explicit up front that the method is returning a function, but it looks like that's the only use of lodash in this directive.